app/assets/javascripts/application.js

```diff
@@ -1,2 +1,11 @@
 //= require_tree ./modules
 //= require govuk_publishing_components/all_components
+//= require set-ga-client-id-on-form
+
+jQuery(function ($) {
+  var $form = $('.js-service-sign-in-form')
+
+  if ($form.length) {
+    new GOVUK.SetGaClientIdOnForm({ $form: $form })
+  }
+})
```

Comment:
It looks like the module system in the `govuk_frontend_toolkit` would be able to handle this. https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#modules

That's used elsewhere in this app, in the `track-radio-group.js` file. Is that something you considered?

Reply:
Yes I can do, I'd need to roll the behaviour into the track-radio-group module since that module is already applied to the form at https://github.com/alphagov/government-frontend/pull/1483/files#diff-1e002ab60ce763d5443df324a62b2bcfR17, unless there's a way to apply two modules to the same element?

Reply:
Hmm, I didn't anticipate a one module per element restriction, but it looks from the code like that's the case [1].

1: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/modules.js#L31

In which case, I'd just stick with your current approach.

Reply:
OK, will do.